src/app/screens/Home/components/CheckBox/styles.js

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+
+export const checkboxStyle = StyleSheet.create({
+  box: {
+    height: 20,
+    width: 20,
```

Comment:
extracto to a const like `const boxSize = 20` between import and export (line 2)